lib/dune

```diff
@@ -2,4 +2,4 @@
  (name Encore)
  (public_name encore)
  (flags (:standard -w -32))
- (libraries butils ke angstrom fmt ocplib-endian ocplib-endian.bigstring))
+ (libraries butils ke angstrom fmt))
```

Comment:
I think `bigstringaf` is missing from this list..

Reply:
Should be fixed with: https://github.com/mirage/encore/commit/34caf3aceb3341a3daffa252b424ead85de7b13c